packages/nimble-components/src/table/components/group-row/template.ts

```diff
@@ -39,9 +39,9 @@ export const template = html<TableGroupRow>`
                 class="expand-collapse-button"
                 tabindex="-1"
                 title="${x => (x.expanded ? tableGroupCollapseLabel.getValueFor(x) : tableGroupExpandLabel.getValueFor(x))}"
+                aria-hidden="true"
```

Comment:
What's the rationale for `aria-hidden="true"`? Could you add a note to the PR description? Is there an accessible interaction or does that need to wait for #1137?

Reply:
The rationale is simply that these buttons are not currently focusable and should thus be hidden from the accessibility tree. As I noted in the new description, this will be temporary for the expand-collapse button in the `Row` component due to the issue you linked.

I'll note that as a result of this change the Accessibility score in Lighthouse drops a couple of points (to 82), seemingly because the `button` element that is within the shadow root of our `Button` element (via the FAST template) still has a non-negative `tabindex`. Through experimentation, if you force that `button` element to have a `tabindex` of -1, the Lighthouse score improves. That said, it's not clear what the right approach to set the `tabindex` on that `button` element is (nor if that's actually what we should be doing).

Reply:
I've update the [linked issue](https://github.com/ni/nimble/issues/1137) with this information.